dataactvalidator/config/awardFields.csv

```diff
@@ -20,8 +20,8 @@ BusinessFundsIndicator,businessfundsindicator,business_funds_indicator,FALSE,str
 BusinessFundsIndicatorDescriptionTag,businessfundsindicatordescriptiontag,business_funds_ind_desc,FALSE,str,,
 BusinessTypes,businesstypes,business_types,FALSE,str,3,
 BusinessTypesDescriptionTag,businesstypesdescriptiontag,business_types_desc,FALSE,str,,
-CFDA_Number,cfda_number,cfda_number,TRUE,str,7,
-CFDA_Title,cfda_title,cfda_title,FALSE,str,100,
+AssistanceListingNumber,assistance_listing_number,assistance_listing_number,TRUE,str,7,
+AssistanceListingTitle,assistance_listing_title,assistance_listing_title,FALSE,str,100,
```

Comment:
There are no spaces in these words so the second columns should be `assistancelistingnumber` and assistancelistingtitle`